source/docs/articles/local/drush-on-pantheon.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How Drush Works on Pantheon
+description: Learn how to create a policy file and sitealias alter hook.
+keywords: Drupal drush, command line, drupal, terminus drush, cli
+---
+One thing that makes Drush site aliases so powerful is the fact that they are defined in regular PHP executable files.  This gives you many options, as the alias definitions can easily be adjusted in code, factoring out common sections, or making other adjustments directly in the alias file.  
+
+* Pantheon automatically generates Drush site alias files for every site available to you. It is preferable to not edit the generated file, so that the local can be easily updated with a pristine copy any time the set of available sites changes.
+
+* Future versions of Drush will be moving away from using PHP executable files for site aliases, and will instead encourage the use of YML files for alias definitions.  This eliminates the concern that a shared alias file might contain code that causes unpredictable side effects.
+
+* Separating code from data is simply a best practice.
+
+## Detect Your Drush Version
+
+Start by running `drush version` to see which version you have installed. To get more information on your Drush configuration, run `drush status`.
+
+## Create a Policy File
+Drush provides a hook that allows you to separate code from data. The mechanism for altering Drush behavior is to use a policy file.  A policy is a regular Drush command file that contains hook implementations rather than Drush commands.
+
```

Comment:
@nataliejeremy what do you think about replacing the doc's intro paragraph with the above paragraph? (Keeping the bullet list)